redis_websocket_api/server.py

```diff
@@ -26,13 +26,16 @@ def __init__(
         """Set default values for new WebsocketHandlers.
 
         :param redis: aioredis.client.Redis instance
-        :param read_timeout: Timeout, after which the websocket connection is
-           checked and kept if still open (does not cancel an open connection)
         :param keep_alive_timeout: Time after which the server cancels the
            handler task (independently of it's internal state)
         """
 
-        self.read_timeout = read_timeout
+        if read_timeout:
+            logger.warning(
+                "read_timeout is not used anymore because cleanup is trigered "
+                "immidiatly on connection loss"
```

Comment:
```suggestion
                "read_timeout is not used anymore because cleanup is triggered "
                "immediately on connection loss"
```